src/meta_schedule/database/memory_database.cc

```diff
@@ -68,20 +68,20 @@ class MemoryDatabaseNode : public DatabaseNode {
     std::vector<std::pair<double, TuningRecord>> results;
     results.reserve(records.size());
     for (const TuningRecord& record : records) {
-      if (!record->run_secs.defined()) {
-        continue;
-      }
-      Array<FloatImm> run_secs = record->run_secs.value();
-      if (run_secs.empty()) {
+      auto run_secs = record->run_secs;
+      if (!run_secs.defined() || run_secs.value().empty() ||
+          std::all_of(run_secs.value().begin(), run_secs.value().end(),
+                      // 1e10 is used as a stub for undefined measurement times.
+                      [](tvm::FloatImm v) { return v.defined() && v->value == 1e10; })) {
```

Comment:
Probably we can define this constant in some common place and reuse it here and in other places?
E.g. as it was done [here](https://github.com/apache/tvm/blob/main/src/meta_schedule/utils.h#L456).

Reply:
I thought about using SortTuningRecordByMeanRunSecs, but memory_database is not sorted unlike json_database.
This constant appears in all tuners and unfortunately there is no common definition for it.
I can suggest to make an NFC patch with refactoring this constant throughout the repository.